Std/Data/Nat/Lemmas.lean

```diff
@@ -1266,3 +1256,6 @@ instance decidableExistsLT [h : DecidablePred p] : DecidablePred fun n => ∃ m
 instance decidableExistsLE [DecidablePred p] : DecidablePred fun n => ∃ m : Nat, m ≤ n ∧ p m :=
   fun n => decidable_of_iff (∃ m, m < n + 1 ∧ p m)
     (exists_congr fun _ => and_congr_left' Nat.lt_succ_iff)
+
+@[deprecated] protected alias lt_connex := Nat.lt_or_gt_of_ne
+@[deprecated] alias pow_two_pos := Nat.two_pow_pos -- deprecated 2024-02-09
```

Comment:
Not sure where these came from but I don't think the first one is necessary anymore. The second could stay for a bit since the comment makes it look recent but I have no idea where it came from.